app/models/episode.rb

```diff
@@ -244,15 +287,32 @@ def path
   end
 
   def include_in_feed?
-    !media? || media_ready?
+    (segment_count.nil? && !media?) || media_ready?
```

Comment:
Bit nuanced - but if you set a non-nil segment_count, you need to actually have that many files.

Reply:
I agree, but I think we need to divorce this from CMS first?  I've been struggling to not add validations that will break incoming CMS announcements.  Like this one, which is probably `validate :published_or_scheduled_episodes_match_segment_count`?

(As I type this, I'm now wondering if this change was too heavy-handed, for CMS updates...)